src/SanityCheck.cpp

```diff
@@ -17,6 +17,7 @@
 #include <string>
 #include <fstream>
 #include <iostream>
+#include <omp.h>
```

Comment:
Add "#ifdef _OPENMP", or remove if we don't need the omp loops down here.

Reply:
Removed in the most recent commit